src/token/types.ts

```diff
@@ -13,9 +13,32 @@ export type Token = {
   symbol: string; // A ticker symbol or shorthand, up to 11 characters
 };
 
+// The raw response from the Swap API
+export type RawTokenData = {
+  name: string;
+  address: Address;
+  currencyCode: string;
+  decimals: number;
+  imageURL: string;
+  blockchain: string;
+  aggregators: string[];
+  swappable: boolean;
+  unverified: boolean;
+  chainId: number;
```

Comment:
How `aggregators`, `swappable`, and  `unverified` work? And should we expose in the Token type?

Reply:
`aggregators` can be ignored, `swappable` is whether the asset can be swapped, and `unverified` - believe there is some sort of verification process for tokens, I'm unsure if this is on the Coinbase side.

I can ask the Wallet team for more info

Reply:
should `swappable` potentially being exposed as `isSwappable: boolean`? it seems a useful data point.

Reply:
yes, but let's leave that for a future PR. let me double check with wallet team **_when_** assets are not `swappable` and we can add that to the documentation. i would assume most assets are by default, swappable

Reply:
sg